### DISCUSSION TITLE
TensorRT support ?

### DISCUSSION THREAD

**Discussion by dmue-vbg** [lllyasviel/stable-diffusion-webui-forge#416] [Q&A]
if i may ask, is webui forge supporting RTX Tensor Units by default or must it be "unleashed" 
by extensions like https://github.com/NVIDIA/Stable-Diffusion-WebUI-TensorRT ? 
Thanks for reply !

**Comment by catboxanon** [comment]
This has been addressed in https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/64#issuecomment-1945960297 -- currently no ETA.
